api/Element.json

```diff
@@ -2900,7 +3026,7 @@
               {
                 "version_added": "10",
                 "version_removed": "65",
-                "alternative_name": "onmozfullscreenchange"
+                "alternative_name": "mozfullscreenchange"
```

Comment:
Is this change correct? This subfeature is under "onfullscreenchange" not "fullscreenchange_event"